scripts/backlog-1105/update-by-uuid.sh

```diff
@@ -0,0 +1,48 @@
+#! /bin/bash
+
+# Given a list of uuids update records in geonetwork
+
+# $1 : show, trial, update, reindex (required)
+# $2 : file containing list of uuids (required)
+# $3 : database host OR geonetwork host (required)
+# $4 : database user
+# $5 : database password
+# $6 : transform xsl file
+
+action="$1";
+
+for uuid in $(cat "$2")
+  do
+    case "$action" in
+      show)
+        # Display the records to be changed
+        java -jar ../../mafia/target/mafia-1.0.0.jar \
+            -url jdbc:postgresql://"$3"/geonetwork_aodn \
```

Comment:
add `geonetwork_aodn` to $3 and your on a winner

Reply:
ok done that